src/main/java/duke/application/Application.java

```diff
@@ -0,0 +1,149 @@
+package duke.application;
+
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.message.Message;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import javafx.application.Platform;
+import javafx.stage.Stage;
+
+/**
+ * The application that manages UI and the taskList.
+ */
+public class Application extends javafx.application.Application {
+    private static final String name = "Iris";
+    private static final String filePath = "C:\\Users\\ortt2\\Documents\\ip\\src\\data\\tasks.txt";
```

Comment:
Perhaps changing the naming to use all uppercase with underscore would follow the coding standard better?

```suggestion
    private static final String NAME = "Iris";
    private static final String FILE_PATH = "C:\\Users\\ortt2\\Documents\\ip\\src\\data\\tasks.txt";
```